core/modules/layout/layout.module

```diff
@@ -1796,3 +1916,368 @@ function layout_get_title_description(Layout $layout) {
 
   return $description;
 }
+
+/**
+ * Store changes to a flexible layout template in the temporary store.
+ *
+ * @param $flexible_template
+ *   The loaded flexible template object.
+ */
+function layout_set_flexible_tempstore($flexible_template) {
```

Comment:
Maybe this could be `layout_flexible_tempstore_set()` to match `layout_flexible_tempstore_load()`.

Reply:
Thanks that's better. Changed.